src/styles/_spotlight.scss

```
@@ -0,0 +1,89 @@
+// AI Spotlight (#576) — AI / チュートリアルが操作した UI 要素を一時的に光らせる。
+// 見た目を全コンポーネントで揃えるため、視覚仕様の source of truth はここ 1 箇所。
+// 色 (--nd-spotlight / --nd-spotlightAlt = テーマの warning 色に連動) と
+// keyframes (nd-spotlight-*) は src/styles/global.css にある。
+// 持続時間 2.4s は useSpotlight.ts の DEFAULT_DURATION_MS と対応。
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)